api/api.py

```diff
@@ -258,11 +261,20 @@ def _format(route):
 ]
 
 
+def custom_json_serializer(obj):
+    if isinstance(obj, bson.objectid.ObjectId):
+        return str(obj)
+    elif isinstance(obj, datetime.datetime):
+        return pytz.timezone('UTC').localize(obj).isoformat()
+    elif isinstance(obj, Job):
+        return obj.map()
+    raise TypeError(repr(obj) + " is not JSON serializable")
```

Comment:
At the time, `Job` imported `util`, which could not in turn import `Job`.

In the most recent commit, that's no longer required, but it stands that any file which uses utility methods could not export a class for json serialization.